### PR TITLE
It's amazing

### PR DESCRIPTION
**Pull request by davidknezic**
Easter egg by @TheDadi! To see, built it yourself!

Made another block instead of integrating it into the `spinner`, because it required slightly different markup. I suggest we adjust the original `spinner` block to the new one at a later point in time.

Also made an alternative to the `ie9-spinner` plugin, named `spinner`.
It finds all elements with `data-default` and `data-fallback` and sets an `is-disabled` or `is-enabled` class, respectively. It does this if the necessary animation features are not supported (ie9).

### CODE REVIEW

**Comment by elessar-ch** [comment]
I'll take a look and merge it tomorrow, when I'm at school....
________________________________

**Comment by digisz** [comment]
should be fluid now ;)

**Comment by davidknezic** [comment]
@digisz Nice!

**Comment by davidknezic** [comment]
We should probably hide this by default. As @sventschui mentioned, there are probably people who don't get this as a joke.

Thoughts? @elessar-ch @digisz @TheDadi @sventschui

**Comment by digisz** [comment]
make it only available by search?

**Comment by davidknezic** [comment]
Yaay.. let me introduce the `cheat` plugin that solves our problem here!

Put the following on you element:

```jade
section.easter-egg(data-cheat).
  You made it!
```

Now try the [Konami code](https://en.wikipedia.org/wiki/Konami_Code), which goes :arrow_up: :arrow_up: :arrow_down: :arrow_down: :arrow_left: :arrow_right: :arrow_left: :arrow_right: :b: :a:

Now to see the amazing bouncing and rolling loading indicator, search for `easter egg`, land on the activity indicators component page and enter the above code. Enjoy :tada: 

@sventschui @elessar-ch Who's gonna merge this?

File: docs/less/blocks/easter-egg.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import 'style/variables';
+
+.easter-egg {
```
Comment (sventschui) [inline docs/less/blocks/easter-egg.less]: why not display: none; ?

Comment (davidknezic) [inline docs/less/blocks/easter-egg.less]: The `display` property isn't animatable. That's why.

**Pull request merged**